app/src/playStore/java/org/fossasia/susi/ai/chat/YouTubeActivity.kt

```diff
@@ -33,8 +33,9 @@ class YouTubeActivity : YouTubeBaseActivity(), YouTubePlayer.OnInitializedListen
         super.onCreate(savedInstanceState)
         setContentView(R.layout.youtube_player)
 
+        val YOUTUBE_KEY = "com.google.android.youtube.API_KEY"
```

Comment:
Make this a private const field

Reply:
@iamareebjamal for this should I make a companion object or top level field?

Reply:
No, just a field